ansible_collections/arista/avd/roles/eos_designs/doc/l3ls-evpn/fabric-topology.md

```diff
@@ -94,6 +94,8 @@ type: overlay-controller
 ```yaml
 # Defined in FABRIC.yml
 
+max_spines: < integer >= number of spine nodes | default spine.nodes | length >  # maximum number of spines, changing this parameter affects address allocation.
```

Comment:
We could describe a use case where this knob could be used and not just default value.